src/transcoding/transcode_ext_handler.cc

```diff
@@ -51,7 +51,7 @@
 #endif
 
 std::unique_ptr<IOHandler> TranscodeExternalHandler::serveContent(const std::shared_ptr<TranscodingProfile>& profile,
-    fs::path location, const std::shared_ptr<CdsObject>& obj, const std::string& range)
```

Comment:
You remove the side effect that location points to the temp fifo after the call. I'm quite sure this is needed for everything to work.

Reply:
wouldn't that need a non const reference?

Reply:
Good point 😄